.appveyor.yml

```diff
@@ -65,6 +65,7 @@ for:
     - "choco install dep"
     - setx PATH "C:\go\bin;C:\gopath\bin;C:\Program Files (x86)\Bazaar\;C:\Program Files\Mercurial;%PATH%;"
     - "go version"
+    - "go env -w GO111MODULE=auto"
```

Comment:
Is this trying to solve the same problem as this? https://github.com/aws/aws-sam-cli/pull/2646
If so maybe we should keep them consistent

Reply:
It seems not the same problem. However, we might need to update the go dep integration test cases to be future-proof.
https://blog.golang.org/go116-module-changes